seed/challenges/05-apis-and-microservices/basic-node-and-express.json

```diff
@@ -61,9 +61,7 @@
         "In the first two lines of the file myApp.js you can see how it’s easy to create an Express app object. This object has several methods, and we will learn many of them in these challenges. One fundamental method is app.listen(<port>). It tells your server to listen on a given port, putting it in running state. You can see it at the bottom of the file. It is inside comments because for testing reasons we need the app to be running in background. All the code that you may want to add goes between these two fundamental parts. HyperDev stores the port number in the environemet variable process.env.PORT. Its value is 3000.",
         "Let’s serve our first string ! In Express, routes takes the following structure: app.METHOD(PATH, HANDLER). METHOD is an http method in lowercase. PATH is a relative path on the server (it can be a string, or even a regular expression). HANDLER is a function that Express calls when the route is matched.",
         "Handlers take the form function(req, res) {...}, where req is the request object, and res is the response object. For example, the handler",
-        "<code>function(req, res) {</code>",
-        "<code> res.send('Response String');</code>",
-        "<code>}</code>",
+        "<blockquote>function(req, res) {<br> res.send('Response String');<br>}</blockquote>",
```

Comment:
@JosephLivengood Hey Joseph, actually, wrapping it in a blockquote and adding the class that I've added to the challenge description wrapper is what will give it the appearance of how code examples look in other challenges. Even though they appear to be wrapped in pre at first glance, stylized blockquotes are the FCC standard. See the code [here](https://github.com/freeCodeCamp/freeCodeCamp/blob/staging/client/less/challenge.less#L75) to check how the effect is achieved.